src/components/Line.tsx

```diff
@@ -0,0 +1,18 @@
+import classnames, {
+  backgroundImage,
+  gradientColorStops,
+  height,
+  width,
+} from 'classnames/tailwind'
+
+const line = (toLeft?: boolean) =>
+  classnames(
+    height('h-px'),
+    width('w-36'),
+    backgroundImage(toLeft ? 'bg-gradient-to-l' : 'bg-gradient-to-r'),
+    gradientColorStops('from-primary-dark', 'to-secondary')
+  )
+
+export default ({ toLeft }: { toLeft?: boolean }) => {
```

Comment:
should we call it `gradientDirection: 'to-left' | 'to-right'`?

Reply:
I think that makes more sense instead of just `toLeft`

Reply:
Changed it. Thanks